commands/releases/info.js

```diff
@@ -14,31 +15,56 @@ function* run (context, heroku) {
     let releases = yield heroku.request({
       path: `/apps/${context.app}/releases`,
       partial: true,
-      headers: {Range: `version ..; max=2, order=desc`},
+      headers: {
+        Range: `version ..; max=2, order=desc`,
+        Accept: `application/vnd.heroku+json; version=3.release_status`
+      },
     });
     id = releases[0].version;
   }
-  release = yield heroku.request({
-    path:    `/apps/${context.app}/releases/${id}`,
-    headers: {Accept: 'application/json'},
-  });
+
+  release = yield {
+    v3: heroku.request({
+      path:    `/apps/${context.app}/releases/${id}`,
+      headers: {
+        Accept: `application/vnd.heroku+json; version=3.release_status`
+      }
+    }),
+    // TODO: move to use API V3 once an endpoint to fetch a release config vars is available
+    v2: function(callback) {
+      heroku.request({
+        path:    `/apps/${context.app}/releases/${id}`,
+        headers: {Accept: 'application/json'},
+        parseJSON: false
+      }, function(err, c) {
+        callback(null, JSON.parse(c || '{}'));
+      });
+    }
+  };
+
   if (context.flags.json) {
-    cli.styledJSON(release);
+    cli.styledJSON(release.v3);
```

Comment:
annoying that this will leave the config vars out of the json, but I suppose we'll have to live with that for now